promql/promqltest/README.md

```diff
@@ -106,8 +106,40 @@ eval range from <start> to <end> step <step> <query>
 * `<start>` and `<end>` specify the time range of the range query, and use the same syntax as `<time>`
 * `<step>` is the step of the range query, and uses the same syntax as `<time>` (eg. `30s`) 
 * `<expect>`(optional) specifies expected annotations, errors, or result ordering.
+* `<expect range vector>` (optional) for an instant query you can specify expected range vector timestamps
 * `<series>` and `<points>` specify the expected values, and follow the same syntax as for `load` above
 
+Note - a string literal instant series is now suppported. The string literal can be defined by placing the string within quotes (""). 
```

Comment:
[nit] I would document what is supported, rather than what's changed:

```suggestion
String literals results are supported. The string literal can be defined by placing the string within quotes (""). 
```